ecc/ecc-stack.c

```diff
@@ -0,0 +1,44 @@
+#define WOLFSSL_TRACK_MEMORY
+#define HAVE_STACK_SIZE
+
+#include <wolfssl/options.h>
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/wolfcrypt/ecc.h>
+#include <wolfssl/wolfcrypt/random.h>
+
+#include <wolfssl/ssl.h>
+#include <wolfssl/test.h>
+
+static RNG rng;
+static ecc_key genKey;
+
+void* do_it(void* args)
```

Comment:
args unused, consider removing input or do a void cast to avoid unused variable warnings in some compilers.

Reply:
To be super-nit-picky consider prototyping ```void* do_it(void);``` before implementation.